.github/workflows/execute-tests-and-promote.yml

```diff
@@ -55,6 +56,16 @@ jobs:
         shell: bash
         run: |
           make generate
+      - name: "Update dependency information after dependabot change"
+        uses: datawire/go-mkopensource/actions/save-dependabot-changes@v0.0.4
+        id: changed-by-dependabot
+        with:
+          branches_to_skip: master
```

Comment:
Branch protection should prevent this but good to be explicit that we don't want any automated commits landing on master with our current setup.